eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToDbsnpVariantsWrapperProcessorTest.java

```diff
@@ -45,6 +60,12 @@
 
     private static SubSnpNoHgvsToDbsnpVariantsWrapperProcessor processor;
 
+    private static final String DECLUSTERED_ALLELES_AND_TYPE_MISMATCH = "Declustered (Alleles and type mismatch)";
+
+    private static final String DECLUSTERED_ALLELES_MISMATCH = "Declustered (Alleles mismatch)";
+
+    private static final String DECLUSTERED_TYPE_MISMATCH = "Declustered (Type mismatch)";
+
```

Comment:
If you are going to check that the constants match, make them public and compare against them directly. Otherwise just a single character change will break the tests.